packages/neovim-nightly/build.sh

```diff
@@ -6,6 +6,7 @@ TERMUX_PKG_MAINTAINER="Aditya Alok <alok@termux.org>"
 # "0.8.0-dev-698-g1ef84547a" < "0.8.0-dev-nightly-10-g1a07044c1", we need to bump
 # the epoch of the package version.
 TERMUX_PKG_VERSION="1:0.8.0-dev-1202-g33dd917d7"
+TERMUX_PKG_REVISION=1
```

Comment:
I added a patch to revert the change that `neovim` should use `libvterm=0.3` yesterday. As for now, this patch is useless, I think we should remove it.